test.d

```diff
@@ -187,24 +184,26 @@ private:
 
     bool libclangExists (const ref Clang clang)
     {
-        auto libclangPath = Path.join(workingDirectory, clang.versionedLibclang);
-        return Path.exists(libclangPath);
+        auto libclangPath = buildNormalizedPath(workingDirectory, clang.versionedLibclang);
+        return exists(libclangPath);
     }
 
     void download (const ref Clang clang)
     {
         auto url = clang.baseUrl ~ clang.filename;
         auto dest = archivePath(clang.filename);
 
-        if (!Path.exists(dest))
-            Http.download(url, dest);
+        import std.net.curl;
```

Comment:
IIRC, Adam (or someone) showed a really simple code snippet in the forums how to download a file over HTTP. I think it was one of those discussions about the dependency on libcurl. I mentioned I use Tango, he showed that snippet. I'm trying to find it...